### ISSUE TITLE
Able to build the image?

### ISSUE THREAD

**Issue by dalareo** [dgamanenko/docker-openedx#1]
I am having a lot of problems building the Dockerfile. Have you successfully built it?

**Comment by Mallfurion** [comment]
@dgamanenko yes, and maybe could you provide a walkthrough for building and deploying open edx with docker?

**Comment by dalareo** [comment]
I finally found this project that did the trick! https://github.com/OpenSaasAU/edx-docker-ansible
